helpers/gogdb_helper.py

```diff
@@ -2,11 +2,33 @@
 import json
 import logging
 import os
+import pathlib
 import sqlite3
+from functools import cmp_to_key
 
-from .constants import PLATFORMS
 from helpers.misc_helper import get_slug_from_title
-from functools import cmp_to_key
+
+from .constants import PLATFORMS
+
+
+def is_sqlite3(path: str) -> bool:
+    """Fast check to ensure a file is indeed an SQLite3 db file, based on the format."""
+    # Thanks! https://stackoverflow.com/a/15355790/895739
+
+    test_sql_file = pathlib.Path(path)
+
+    if not test_sql_file.exists():
+        return False
+    if not test_sql_file.is_file():
+        return False
```

Comment:
`is_file` will be `False` if the file doesn't exist, so no need for the `exists` test unless I'm missing something.

```
>>> os.path.isfile("poo")
False
```

Reply:
What if it is a directory...

Reply:
Then `is_file()` will be `False` as well. I'm saying to get rid of `exists()`, not `is_file()`.

Reply:
Ahhh... smart.